src/main/java/org/andersen/homework/model/dto/ticket/TicketDto.java

```diff
@@ -0,0 +1,17 @@
+package org.andersen.homework.model.dto.ticket;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.andersen.homework.model.dto.user.client.ClientIdOnlyDto;
+
+@Getter
+@Setter
+@ToString
+public class TicketDto {
+  private UUID id;
+  private ClientIdOnlyDto client;
```

Comment:
Why we need this Dto?

Reply:
If I use `ClientDto` here, which contains TicketDto, infinite recursion will appear and gain an exception, but `ClientIdOnlyDto` contains only client id and does not lead to throwing exceptions.

Reply:
Actually my question is why do we need separate ClientIdOnlyDto with only 1 field inside and don't use just UUID here :)

Reply:
And field would be private UUID clientId;